File: seed/go-sdk/path-parameters/inline-path-parameters/dynamic-snippets/example2/snippet.go
package example

import (
    client "github.com/fern-api/path-parameters-go/client"
    context "context"
    path "github.com/fern-api/path-parameters-go"
)

func do() () {
    client := client.NewClient()
    client.Organizations.SearchOrganizations(
        context.TODO(),
        "tenant_id",
        "organization_id",
        &path.SearchOrganizationsRequest{
            Limit: path.Int(
                1,
            ),
        },
    )
}

File: seed/ts-sdk/path-parameters/retain-original-casing/src/dynamic-snippets/example1/snippet.ts
import { SeedPathParametersClient } from "../..";

async function main(): Promise<void> {
    const client = new SeedPathParametersClient({
        environment: "https://api.fern.com",
    });
    
    await client.organizations.getOrganizationUser("tenant_id", "organization_id", "user_id");
}
main();
